src/components/molecules/widgets/CountBySeverityTextWidget.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { FC } from 'react';
 import { IWidgetCountBySeverityTextData } from '../../../models/WidgetData';
 import TextView from '../TextView/TextView';
+import RootStore from '../../../store/root.store';
+import { useStore } from '../../../store/storeConfig';
 
 interface IProps {
   data: IWidgetCountBySeverityTextData;
-  segmentText: string;
 }
-const CountBySeverityTextWidget: FC<IProps> = ({ data, segmentText }) => {
-  return <TextView data={data} segmentText={segmentText} />;
+const CountBySeverityTextWidget: FC<IProps> = ({ data }) => {
+  const store : RootStore = useStore();
```

Comment:
molecules are NOT store awared!

Reply:
can I pass `store.newsFlashWidgetsMetaSegmentName` as a prop to WidgetWrapper from the widget Template?

Reply:
yes - See how it is done in other components.